_overviews/FAQ/index.md

```diff
@@ -155,7 +154,7 @@ and the [typeclass pattern](http://tpolecat.github.io/2013/10/12/typeclass.html)
 
 This [blog post](http://tpolecat.github.io/2015/04/29/f-bounds.html)
 argues against F-bounds and in favor of typeclasses.
-Which is revisited with a discussion about the trade-offs of alternatives
+Another discussions of trade-offs and alternatives is
```

Comment:
I mean to say that the SO post discuss trade-offs between all alternatives.
While I like your wording more than mine, I am not totally sure if it gives the correct idea.

Reply:
I pushed another commit with an adjustment; wdyt?

Reply:
I like how you have marked your link answers as "community wiki", btw 👍

Reply:
@SethTisue It looks good to me!